frontstage/views/account/account_survey_share.py

```diff
@@ -149,17 +158,16 @@ def build_payload(respondent_id):
         }
     """
     email = flask_session['share_survey_recipient_email_address']
-    business_id = flask_session['share_survey_business_selected']
-    share_survey_surveys_selected = flask_session['share_survey_surveys_selected']
     payload = {}
     pending_shares = []
-    for survey in share_survey_surveys_selected:
-        pending_share = {'business_id': business_id,
-                         'survey_id': survey,
-                         'email_address': email,
-                         'shared_by': respondent_id}
-        pending_shares.append(pending_share)
-
+    share_dictionary = flask_session['share']
+    for key in share_dictionary:
```

Comment:
Can key be changed to what it is?  I think it's a business_id so it should be called that

Reply:
👍